obspyDMT/utils/data_handler.py

```diff
@@ -486,8 +486,10 @@ def arc_serial_parallel(stas_avail, event, input_dics, target_path,
     """
     print('%s -- event: %s' % (req_cli, target_path))
 
-    client_arclink = Client_arclink(user='test@obspy.org',
-                                    timeout=input_dics['arc_wave_timeout'])
+    client_arclink = Client_arclink(user=input_dics['username'],
+                                    password=input_dics['password'],
+                                    timeout=input_dics['arc_avai_timeout'])
```

Comment:
Maybe we should replace `input_dics['arc_avai_timeout']` with `input_dics['arc_wave_timeout']` here as the latter is:

`input_dics['arc_wave_timeout']`: Timeout (in sec) for sending a waveform data or metadata request via ArcLink (default: 2).

Reply:
Isn't the parameter *'arc_avai_timeout'* meant for the timeout settings of the function *'arc_available'* in **metadata_handler.py** for checking data availability only?
Furthermore, I can not find another usage of *'arc_wave_timeout'* than in **data_handler.py**, where the client is used to get waveform data or metadata (as described in the documentation).

Reply:
Yes, exactly, the parameter ```arc_avai_timeout``` is for the timeout settings of the function ```arc_available``` in ```metadata_handler.py```. This is why I suggested changing ```input_dics['arc_avai_timeout']```  to ```input_dics['arc_wave_timeout']``` here (note that this comment is for the changes in ```data_handler.py``` and not ```metadata_handler.py```). Does it make sense?

Reply:
Oh I see! I thought you meant the opposite, but I obviously made a copy paste error as I did not notice the difference between `arc_wave_timeout` and `arc_avai_timeout` in the two functions. My bad!

Reply:
Thanks for the changes.